### PR TITLE
 SPST-42 Fix flow for the page component

### PR DESCRIPTION
**Pull request by maniator**
Every time the flow defs are recreated they come out in a different order, that is just how the library we are using reads the ast (it seems to always be in a different order) 

The main changes are in the types of the page component

### CODE REVIEW

File: flow/plasma_vx.x.x.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// flow-typed version: <<STUB>>/@wework-dev/plasma_v0.1.4/flow_v0.105.2
+// flow-typed version: <<STUB>>/@wework-dev/plasma_v0.1.5-alpha.0/flow_v0.105.2
```
Comment (GeReV) [inline flow/plasma_vx.x.x.js]: You changed package.json to 0.1.6.

Comment (maniator) [reply flow/plasma_vx.x.x.js]: Will be updated when the whole thing is rebuilt in a separate PR

File: src/components/layout/Page/Page.jsx [hunk 2]
```diff
@@ -6,10 +6,10 @@ import type { Data } from '../../../types';
 import style from './style.scss';
 
 type Props = {|
-  title: string,
+  title?: string,
   children: React$Node,
-  actions: Array<Object>,
-  breadcrumb: React$Node,
+  actions?: Array<Object>,
+  breadcrumb?: React$Node,
```
[the remark this answers is not in the document]
Comment (shem8) [reply src/components/layout/Page/Page.jsx]: so why the other changes? =?

[the remark this answers is not in the document]
Comment (maniator) [reply src/components/layout/Page/Page.jsx]: That is how jscode that we are using generates the flow file.

It always seems to do it in a different order

We can try to fix that in the future (see PR description)

**Pull request merged**